src/test/shell/bazel/remote/remote_execution_test.sh

```diff
@@ -3204,4 +3204,72 @@ EOF
   expect_not_log "WARNING: Writing to Remote Cache:"
 }
 
+function test_download_toplevel_when_turn_remote_cache_off() {
+  # Test that BwtB does cause build failure if remote cache is disabled in a following build.
+  # See https://github.com/bazelbuild/bazel/issues/13882.
+
+  mkdir -p a
+  cat > a/BUILD <<EOF
```

Comment:
nit: you can use `EOF` and not need to escape the `$` in the content.

Reply:
Done.